dotnet/samples/Concepts/TextToImage/OpenAI_TextToImage.cs

```diff
@@ -82,32 +82,55 @@ A cute baby sea otter
     }
 
     [Fact]
-    public async Task SimpleTextToImageExampleAsync()
+    public async Task SimpleDallE3ImageUriAsync()
     {
         var builder = Kernel.CreateBuilder()
-           .AddAzureOpenAITextToImage( // Add your text to image service
-               deploymentName: TestConfiguration.AzureOpenAI.ImageDeploymentName,
-               endpoint: TestConfiguration.AzureOpenAI.ImageEndpoint,
-               apiKey: TestConfiguration.AzureOpenAI.ImageApiKey,
-               modelId: TestConfiguration.AzureOpenAI.ImageModelId);
+            .AddOpenAITextToImage( // Add your text to image service
+                modelId: "dall-e-3",
+                apiKey: TestConfiguration.OpenAI.ApiKey);
 
         var kernel = builder.Build();
         var service = kernel.GetRequiredService<ITextToImageService>();
 
-        var generatedImages = await service.GetImageContentsAsync(new TextContent("A cute baby sea otter"), new OpenAITextToImageExecutionSettings { Size = (Width: 1792, Height: 1024) });
+        var generatedImages = await service.GetImageContentsAsync(
+            new TextContent("A cute baby sea otter"),
+            new OpenAITextToImageExecutionSettings { Size = (Width: 1792, Height: 1024) });
 
         this.Output.WriteLine(generatedImages[0].Uri!.ToString());
     }
 
     [Fact]
-    public async Task OpenAIDallE3Async()
+    public async Task SimpleDallE3ImageBinaryAsync()
+    {
+        var builder = Kernel.CreateBuilder()
+            .AddOpenAITextToImage( // Add your text to image service
+                modelId: "dall-e-3",
+                apiKey: TestConfiguration.OpenAI.ApiKey);
+
+        var kernel = builder.Build();
+        var service = kernel.GetRequiredService<ITextToImageService>();
+
+        var generatedImages = await service.GetImageContentsAsync(new TextContent("A cute baby sea otter"),
+            new OpenAITextToImageExecutionSettings
+            {
+                Size = (Width: 1024, Height: 1024),
+                ResponseFormat = "bytes",
+                // ResponseFormat = OpenAI.Images.GeneratedImageFormat.Bytes 
```

Comment:
Commented out code?

Reply:
I left there in purpose because this is also a `object?` and can be provided with the underlying SDK type as well, I can be more verbose on my comments